packages/playground/website/public/gutenberg.html

```diff
@@ -212,6 +214,20 @@
 					},
 				],
 			};
+			// If there's a import-site query parameter, pass that to the blueprint
+			const urlParams = new URLSearchParams(window.location.search);
+			const importSite = new URL(urlParams.get('import-site'));
```

Comment:
This line fails because, importSite isn't a url.

```suggestion
			const importSite = urlParams.get('import-site');
```

Reply:
That's intentional, but I see I didn't catch the error well. Let me try again.